docs/unified/protectli/overview.md

```diff
@@ -170,6 +170,53 @@
         describes the platform's hardware configuration used during the Dasharo
         firmware validation procedure.
 
+=== "VP2430"
+    The VP2430 is a small form network appliance built for use as a firewall
+    / router, virtualization platform, a daily-driven personal computer,
+    and more. A major difference setting it apart from its predecessors is the
+    CPU. While the 2410 and 2420 models both utilize a J-series Intel Celeron,
+    the VP2430 comes equipped with a more recent Alder Lake N-series CPU. This
+    means a significant improvement in performance, at a comparable power
+    consumption level. It also features 4 x 2.5G i226 ethernet ports.
+
+    ![](/images/VP2430.png)
```

Comment:
File not present?

Reply:
[Fixed](https://github.com/Dasharo/docs/pull/932/files#diff-dd560e92537a8f0e93a0c54ae81f1aea0fabf16f9fe13634897e4bfe64c5e08c)